python/jetToolbox_cff.py

```diff
@@ -484,6 +484,25 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
 			toolsUsed.append( 'selectedPatJets'+jetALGO+'PF'+PUMethod+'SoftDropPacked' )
 			toolsUsed.append( 'selectedPatJets'+jetALGO+'PF'+PUMethod+'SoftDropSubjets' )
 
+                        ## Pack fat jets with subjets
+			setattr( proc, 'packedPatJets'+jetALGO+'PF'+PUMethod, 
```

Comment:
added, I also notice that the parameter distMax was set to 0.8 earlier and now 
I change it to 
 distMax= cms.double( jetSize )